vhost-device-vsock/vhost-device-vsock.1

```diff
@@ -0,0 +1,204 @@
+.Dd July 24, 2024
+.Dt VHOST-DEVICE-VSOCK 1
+.Os
+.Sh NAME
+.Nm vhost-device-vsock
+.Nd A virtio-vsock device using the vhost-user protocol
+.Sh SYNOPSIS
+.Nm
+.Fl \-socket Ar SOCKET
+.Fl \-uds-path Ar UDS_PATH
+.Sh DESCRIPTION
+The
+.Nm
+utility launches a daemon listening to
+.Ar SOCKET
+for incoming connections from
+.Em vhost-user
+front-ends.
+Upon successful connection and protocol negotiation,
+it starts a vhost-user backend instance.
+The functionality of a VIRTIO vsock device is provided
+using the specified unix socket,
+.Ar UDS_PATH Ns
+, to which a host-side application connects to.
+.Bl -tag -width Ds
+.It Fl -help | h
+Print help information.
+.It Fl -version | V
+Print version information.
+.It Fl \-guest-cid Pa GUEST_CID
+Context identifier of the guest which uniquely identifies the
+device for its lifetime
+.Bq default: 3
+.It Fl \-socket Pa SOCKET
+Unix socket to which a hypervisor connects to and sets up the
+control path with the device
+.It Fl \-uds-path Pa UDS_PATH
+Unix socket to which a host-side application connects to
+.It Fl \-tx-buffer-size Pa TX_BUFFER_SIZE
+The size of the buffer used for the TX virtqueue
+.Bq default: 65536
+.It Fl \-queue-size Pa QUEUE_SIZE
+The size of the vring queue
+.Bq default: 1024
+.It Fl \-groups Pa GROUPS
+The list of group names to which the device belongs.
+A group is a set of devices that allow sibling communication between their
+guests.
+.Bq default: default
+.It Fl \-config Pa CONFIG
+Load from a given configuration file
+.sp
+.Sy NOTE :
+conflicts with
+.Li --vm
+.It Fl \-vm Pa VM
+Device parameters corresponding to a VM in the form of comma
+separated
+.Li key=value
+pairs.
+Multiple instances of this option
+can be provided to configure devices for multiple guests.
+The allowed keys are:
+.Bl -enum -compact -width 0 -offset 1n
+.It
+.Li guest_cid
+.It
+.Li socket
+.It
+.Li uds_path
+.It
+.Li tx_buffer_size
+.It
+.Li queue_size
+.It
+.Li group
+.El
+.Sy NOTE :
+conflicts with
+.Li --config
+.El
+.Sh EXIT STATUS
+.Ex -std
+.Sh EXAMPLES
+.Ss Run the vhost-device-vsock device
+.Bd -literal
+vhost-device-vsock --guest-cid=<CID assigned to the guest> \\
+  --socket=<path to the Unix socket to be created to communicate with the VMM via the vhost-user protocol> \\
+  --uds-path=<path to the Unix socket to communicate with the guest via the virtio-vsock device> \\
+  [--tx-buffer-size=<size of the buffer used for the TX virtqueue (guest->host packets)>] \\
+  [--queue-size=<size of the vring queue>] \\
+  [--groups=<list of group names to which the device belongs concatenated with '+' delimiter>]
```

Comment:
These lines are a bit long too, and the explanations redundant, I suggest replacing the `<` `>` bits with example values

Reply:
I think that examples with values instead of explanation are already covered. Do you prefer to get this example removed?

Reply:
Done